findimports.py

```diff
@@ -559,13 +562,16 @@ def __init__(self):
         self._module_cache = {}
         self._warned_about = set()
         self._stderr = sys.stderr
-        self._exts = ('.py', '.so', '.dll')
+        self._exts = ['.py', '.so', '.dll']
         if hasattr(sys, '_multiarch'):  # pragma: nocover
             # Ubuntu 14.04 LTS renames
             # /usr/lib/python2.7/lib-dynload/datetime.so to
             # /usr/lib/python2.7/lib-dynload/datetime.x86_64-linux-gnu.so
             # (https://github.com/mgedmin/findimports/issues/3)
-            self._exts += f".{sys._multiarch}.so"
+            self._exts.append(f".{sys._multiarch}.so")
```

Comment:
This looks like a bugfix for an impossible bug.   This was once adding a tuple to a list of tuples, then the tuple got lost during the conversion into f-strings (my bad for not noticing), but since this is Python 2.7-only code that can't possibly run any more (no f-strings on Python 2.7), this entire `if` should just be removed.